cmd/main.go

```diff
@@ -28,6 +28,7 @@ import (
 var version = "undefined"
 
 const historyLength = 256
+const KLAOSChainID = 2718
```

Comment:
Lowercase, please. No need to export this constant

Reply:
changed